Cubical/Data/Int/MoreInts/QuoInt/Properties.agda

```diff
@@ -233,3 +235,82 @@ sign-pos-suc-· m (signed s (suc n)) = refl
 
 ·-injʳ : ∀ m n o → m · pos (suc o) ≡ n · pos (suc o) → m ≡ n
 ·-injʳ m n o p = ·-injˡ o m n (·-comm (pos (suc o)) m ∙ p ∙ ·-comm n (pos (suc o)))
+
+opIsoIntℤ : Iso (Int → Int → Int) (ℤ → ℤ → ℤ)
+opIsoIntℤ = (opIso isoIntℤ)
+
+_+'_ : ℤ → ℤ → ℤ
+_+'_ = Iso.fun opIsoIntℤ _+Int_
+
+sucℤ→Int : ∀ (n : ℤ) → sucInt (ℤ→Int n) ≡ ℤ→Int (sucℤ n)
+sucℤ→Int (pos n) = refl
+sucℤ→Int (neg zero) = refl
+sucℤ→Int (neg (suc zero)) = refl
+sucℤ→Int (neg (suc (suc n))) = refl
+sucℤ→Int (posneg i) = refl
+
+predℤ→Int : ∀ (n : ℤ) → predInt (ℤ→Int n) ≡ ℤ→Int (predℤ n)
+predℤ→Int (pos zero) = refl
+predℤ→Int (pos (suc n)) = refl
+predℤ→Int (neg zero) = refl
+predℤ→Int (neg (suc n)) = refl
+predℤ→Int (posneg i) = refl
+
+ℤ→Int+Int≡+ : ∀ (n m : ℤ) → (ℤ→Int n) +Int (ℤ→Int m) ≡ ℤ→Int (n + m)
+ℤ→Int+Int≡+ (pos zero) m =
+  (ℤ→Int (pos zero)) +Int (ℤ→Int m)
+  ≡⟨ sym (pos0+ (ℤ→Int m)) ⟩
```

Comment:
I'd suggest indenting the reasoning lines with a different number of spaces to improve readability.

Reply:
Hm, I changed the formatting into something else entirely, because it felt nicer. Lemme know what you think and if you want a specific different one then tell me what it should be.